miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -489,6 +490,15 @@ public static SampleDto asDto(@Nonnull Sample from, boolean includeBoxPositions,
       setId(dto::setRequisitionId, requisition);
       setString(dto::setRequisitionAlias, requisition.getAlias());
       dto.setRequisitionAssayIds(requisition.getAssays().stream().map(Assay::getId).toList());
+      dto.setRequisitionStopped(requisition.isStopped());
+      for (RequisitionPause pause: requisition.getPauses())
+      {
+        LocalDate endDate = pause.getEndDate();
+        if (endDate == null || endDate.isAfter(LocalDate.now()))
+        {
+          dto.setRequisitionPaused(true);
```

Comment:
you can `break` out of the loop once you find the first reason to set this to true - the rest is a waste of cycles

Reply:
Alternatively, stream the pauses and use anyMatch()

Reply:
This is also repeated in 5 places. Extract to a private static method